test/e2e/pivoting_based_feature_test.go

```diff
@@ -24,7 +24,7 @@ var (
 	workerMachineCount       int64
 )
 
-var _ = Describe("Testing features in ephemeral or target cluster", func() {
+var _ = Describe("Testing features in ephemeral or target cluster [pivoting] [features]", func() {
```

Comment:
Please keep it consistent. The other two has no space between.

Reply:
Thanks for noticing. Added spaces for other changes as well.